Kozariz.Edelveis.Project/Pages/Shared/_LoginPartial.cshtml

```diff
@@ -0,0 +1,26 @@
+﻿@namespace Edelveis.Db
+@using Microsoft.AspNetCore.Identity
+ 
+@inject SignInManager<Edelveis.Db.Models.ApplicationUser> SignInManager
```

Comment:
The code snippet utilizes the SignInManager and UserManager from the Microsoft.AspNetCore.Identity namespace to conditionally render a navigation menu based on the user's signed-in status, but it lacks proper separation of concerns, as the code for the navigation menu and authentication logic are tightly coupled together in the same file, which can make the code less maintainable and harder to understand and test.

Reply:
I can`t really agree on that point. Separating those concerns will lead to unuseful `navbar`